filebeat/filebeat.yml

```diff
@@ -82,14 +82,14 @@ output.elasticsearch:
   # Array of hosts to connect to.
   hosts: ["localhost:9200"]
 
-  # Template name. By default the template name is filebeat.
-  template.name: "filebeat"
+  # Load the Elasticsearch template.
+  # template.enabled: true
```

Comment:
As it is now loaded by default, I'm not even sure if we still needed in the short config?